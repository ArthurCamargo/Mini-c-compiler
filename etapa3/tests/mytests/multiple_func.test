int f() {
    a = f;
}
int a() {}
