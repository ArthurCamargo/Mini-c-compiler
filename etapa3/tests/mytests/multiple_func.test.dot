digraph G {
n0;
n0 -> n1;
n0 -> n2;
n1;
n1 -> n3;
n1 -> n4;
n3;
n3 [label="a"];
n4;
n4 [label="f"];
n1 [label="="];
n2;
n2 [label="a"];
n0 [label="f"];

}