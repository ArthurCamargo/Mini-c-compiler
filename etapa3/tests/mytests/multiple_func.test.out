0x55ce655fe350
0x55ce655fe350, 0x55ce655fe480
0x55ce655fe350, 0x55ce655fe4f0
0x55ce655fe480
0x55ce655fe480, 0x55ce655fe3c0
0x55ce655fe480, 0x55ce655fe430
0x55ce655fe3c0
0x55ce655fe3c0 [label="a"];
0x55ce655fe430
0x55ce655fe430 [label="f"];
0x55ce655fe480 [label="="];
0x55ce655fe4f0
0x55ce655fe4f0 [label="a"];
0x55ce655fe350 [label="f"];
